appcues/src/main/java/com/appcues/ui/ModalStateMachineOwner.kt

```diff
@@ -113,11 +112,10 @@ internal class ModalStateMachineOwner(
         // cancel any previous
         uiIdleDebounceTimer?.cancel()
 
-        uiIdleDebounceTimer = Timer().schedule(delay = SCROLL_DEBOUNCE_MILLISECONDS) {
-            // set to know to allow new Timer
-            uiIdleDebounceTimer = null
```

Comment:
note: we are losing the setting of this value back to null here, but looking at the code, it seems like we don't actually need that, correct? The prior comment just made it sound like it was needed, but perhaps that was an old comment.

Reply:
it was a typo on the old comment, but its not required